src/components/atoms/BadgePreview/v2/badgePreview.scss

```diff
@@ -1,5 +1,5 @@
-@use '@assets/scss/functions/_base.functions' as *;
-@use '@assets/scss/variables/variables.module';
+@use '../../../../assets/scss/functions/_base.functions' as *;
```

Comment:
its not possible to use alias path here?

Reply:
Nop, the build fails if we use it. I don't want to wast time on it, its something around TS and Rollup.